content/v2/openapi.yml

```diff
@@ -3608,6 +3608,8 @@ components:
           type: boolean
         whois_privacy:
           type: boolean
+        linked_provider:
```

Comment:
We should also update the example. This is the Domain Registration model and not the RegisterDomain endpoint. If I understand correctly it also needs to be updated with the new parameter.

Reply:
Thanks! I fixed this now.

Reply:
@kntsoriano does this mean that we do not return the provider this domain is registered within our domain model? If so have you considered adding it to the model?

My thinking on this is that as an integration developer, I can use the provider nickname to at the very least create a local mapping of Nickname -> GoDaddy, DNsimple, etc. This information could be used to visualize the domains differently or group them in my UI or as a filter for certain operations or policies.

Reply:
At the moment, we don't return it. I think it's a good idea to return the provider nickname inside the domain model :+1: